qiita_pet/handlers/study_handlers/artifact.py

```diff
@@ -107,6 +107,16 @@ def patch(self):
         self.write(response)
 
 
+class ArtifactGetSamples(BaseHandler):
+    @authenticated
+    def get(self):
+        aids = map(int, self.request.arguments.get('ids[]', []))
```

Comment:
Note that I used [self.get_arguments](http://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.get_arguments) instead of [self.get_argument](http://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.get_argument). According to the documention it should return a list of parameters.